org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -276,3 +278,8 @@ Outputs:
     Description: Arn of the findings queue
     Export:
       Name: !Sub '${AWS::StackName}-SecurityHubFindingsQueueArn'
+  SecurityHubFindingsSuppressionLambdaFctName:
```

Comment:
you are taking a passed in value and then just outputting it?  I'm not sure why you need to do this.  Is there some purpose for this output that i'm not aware of?

Reply:
I used to have everything in the same CF stack (like the CDK), but I (over?)thought it'd be better to split the lambda out in case we need to add some logic to it (in which case we'd also move it to SAM)... The param/output is just to pass the lambda function name, which is needed in the policy (https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/4c07059b53167d7ef54132dfd0e91d18e4290693/org-formation/075-security-hub/security-hub-suppress-infra.yaml#L23) but not accessible as an output (output from lambda is either arn or resource name, not function name, besides that'd be a circular dependency as you mentioned in other PR). I can put it all back in one stack...